src/Exception/InvalidException.php

```diff
@@ -7,20 +7,42 @@
  */
 class InvalidException extends AuthException
 {
-    const CODE_TOKEN_EXPIRED = 1;
-    const CODE_TOKEN_INVALID = 2;
-    const CODE_CREDENTIALS_INVALID = 3;
-
     public function __construct(
         $message = 'The token being used is invalid',
-        $code = 0,
         \Exception $previous = null
     ) {
-        parent::__construct($message, $code, $previous);
+        parent::__construct($message, 403, $previous);
```

Comment:
I'd rather not see `__construct` overloaded. It makes things harder to extend and less predictable.